ui/home/Stats.tsx

```diff
@@ -120,6 +121,15 @@ const Stats = () => {
           url={ route({ pathname: '/txs' }) }
           isLoading={ isLoading }
         />
+        { data.last_output_root_size && (
```

Comment:
maybe add a rollup feature enabled condition? otherwise if frontend is not configured (for some reason) /batches is 404